Update sql engineers docs (#684)
SimpleDataLabsInc/prophecy-docs: docs/analysts/development/models/gems/sources-target/sources-target.md

---
title: Sources and targets
id: sources-and-targets
slug: /analysts/model-sources-and-targets
description: Use models to read and write data
tags:
  - SQL
  - sources
  - seeds
---

> Model sources and target vary slightly from pipeline sources and targets. In brief, all models sources and targets must point to tables in the SQL warehouse.

Reviewer: ```suggestion
Model sources and target vary slightly from pipeline sources and targets. The main difference being that all model sources and targets must point to tables in the SQL warehouse.
```
Suggestion: Wording.

## Sources

When you create a new model, you need to define an input data source. The data source can be:

> - [Table gem](/analysts/table). You can browse available tables in the **Environment** tab of the Project Editor.

Reviewer: ```suggestion
- A [Table gem](/analysts/table), which allows you to browse available tables in the **Environment** tab of the Project Editor.
```

> - Another model. Drag a model from the left sidebar onto your canvas to use it as a source.

Reviewer: ```suggestion
- Another model, which you can drag from the left sidebar onto your canvas to use as a source.
```
